src/CADL.Extension/Emitter.Csharp/package.json

```diff
@@ -35,12 +35,12 @@
         "dist/**"
     ],
     "devDependencies": {
-        "@cadl-lang/compiler": "~0.33.0",
-        "@cadl-lang/eslint-config-cadl": "~0.3.0",
-        "@cadl-lang/eslint-plugin": "~0.1.0",
-        "@cadl-lang/library-linter": "^0.1.2",
-        "@cadl-lang/rest": "^0.15.1",
-        "@cadl-lang/versioning": "~0.6.0",
+        "@cadl-lang/compiler": "latest",
```

Comment:
Will it cause build break if CADL libraries contains breaking change?

Reply:
Yes. But we need to ware the changes of the cadl compiler at this moment since cadl compile is not stable. When cadl compile is stable, we will update to a special stable version.

Reply:
My worry is if you enable it into the build pipeline, the CADL library change would cause the entire build failure which impacts others to merge PR.

Reply:
I updated to use stable version of cadl.